### PR TITLE
image cta on website

### PR DESCRIPTION
**Pull request by penk85**
WPC-4348(b): Add the image call to action on website under peer article

### CODE REVIEW

File: examples/website/src/shared/route/articleTemplateContainer.tsx [hunk 1]
```diff
@@ -4,25 +4,25 @@ import {gql, useQuery} from '@apollo/client'
 import {articleAdapter, peerAdapter} from './articleAdapter'
 
 import {
+  articleMetaDataFragment,
```
Comment (bishoymelek) [inline examples/website/src/shared/route/articleTemplateContainer.tsx:7]: these changes or re-arranging is confusing, can you please clean it? @penk85

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 2]
```diff
@@ -109,7 +109,9 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
             setValidPeer(false)
             setErrorMessage(t('peerList.panels.invalidURL'))
           } else if (response?.data?.peerProfile.hostURL === hostURL) {
-            setValidPeer(false)
+            // TODO for testing! To be removed!
```
Comment (bishoymelek) [inline packages/editor/src/client/panel/peerEditPanel.tsx:112]: @penk85 should this be removed?

**Review by bishoymelek — CHANGES_REQUESTED** [review]
Great one @penk85 , I left few simple comments.

File: packages/editor/src/client/panel/peerEditPanel.tsx (hunk 2 above)
Comment (penk85) [reply packages/editor/src/client/panel/peerEditPanel.tsx:112]: Yes it was for Laurent to peer to self while he tests. I'll set it back.

**Review by bishoymelek — DISMISSED** [review]
🎉 great work @penk85

File: packages/editor/src/client/locales/en.json [hunk 3]
```diff
@@ -643,7 +644,8 @@
         "token": "Token",
         "Unknown": "Unknown",
         "URL": "URL",
-        "validateToken": "Please confirm token is correct before saving"
+        "validateToken": "Please confirm token is correct before saving",
+        "leaveEmpty": "Leave empty if you don't want to change it"
       }
```
Comment (camilleory) [inline packages/editor/src/client/locales/en.json:649]: Shouldn't those also come into fr.json and de.json ?

Comment (roosnic1) [reply packages/editor/src/client/locales/en.json:649]: @camilleory translations happen with a tool once the new translation string have been merge to `master`

**Review by roosnic1 — APPROVED** [review]

**Review by camilleory — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]
🎉 💯

**Review by lakeside763 — APPROVED** [review]
This looks good to me

**Pull request merged**